app/services/search_service.rb

```diff
@@ -62,7 +62,12 @@ def url_resource_symbol
 
   def full_text_searchable?
```

Comment:
That's not the right method. Also, def/end blocks can have a rescue clause without extra begin/end.

Reply:
like this?
```ruby
def method
  do_something
rescue Exception
  fallback
end
```

Reply:
Yes, like that. :+1: